apps/passport-server/src/services/generic-issuance/pipelines/CSVPipeline/CSVPipeline.ts

```diff
@@ -333,6 +335,14 @@ export async function makeCSVPCD(
           opts.requesterSemaphoreId,
           opts.pipelineId
         );
+      case CSVPipelineOutputType.PODTicket:
+        return makePODTicketPCD(
```

Comment:
this is probably handled elsewhere but would be good to make sure this can't issue tickets with same product/event ids as other pipelines

Reply:
I think this is handled by having the UUID-v5s namespaced to the pipeline ID.